src/themes.php

```diff
@@ -1358,4 +1358,16 @@
         "sideLabels" => "#8D192B",
         "dates" => "#862931",
     ],
+     "neon" => [
+        "background" => "#000000",
+        "border" => "#ffffff",
+        "stroke" => "#ffffff",
```

Comment:
Minor thing for consistency

```suggestion
        "border" => "#FFFFFF",
        "stroke" => "#FFFFFF",
```